waddrmgr/manager.go

```diff
@@ -2289,26 +2289,26 @@ func Open(namespace walletdb.Namespace, pubPassphrase []byte, chainParams *chain
 //
 // A ManagerError with an error code of ErrAlreadyExists will be returned the
 // address manager already exists in the specified namespace.
-func Create(namespace walletdb.Namespace, seed, pubPassphrase, privPassphrase []byte, chainParams *chaincfg.Params, config *ScryptOptions) (*Manager, error) {
+func Create(namespace walletdb.Namespace, seed, pubPassphrase, privPassphrase []byte, chainParams *chaincfg.Params, config *ScryptOptions) error {
```

Comment:
Looking through the PR, I can see that 8 call sites needed code added to deal with this when realistically only the one for later use in `wallet.Open`.

Instead of deferring then, why not just close the mgr returned from `Create` in the `wallet.Create` function and leave everything else untouched?